src/geometry/manifold/times_cont_mdiff.lean

```diff
@@ -1287,6 +1324,59 @@ lemma smooth_within_at_proj
     (proj I M) s p :=
 basic_smooth_bundle_core.smooth_within_at_proj _
 
+variables (I M)
+/-- The zero section of the tangent bundle -/
+def zero_section : M → tangent_bundle I M := λ x, ⟨x, 0⟩
+variables {I M}
+
+lemma smooth_zero_section : smooth I I.tangent (zero_section I M) :=
+begin
+  apply basic_smooth_bundle_core.smooth_const_section (tangent_bundle_core I M) 0,
+  assume i j x hx,
+  simp only [tangent_bundle_core, continuous_linear_map.map_zero] with mfld_simps
+end
+
+/-- The derivative of the zero section of the tangent bundle maps `⟨x, v⟩` to `⟨⟨x, 0⟩, ⟨v, 0⟩⟩`.
+
+Note that this statement relies on the choice of the coordinate system we use on the tangent bundle.
+However, there is a canonical one at a point of the zero section (since there is a canonical
+horizontal space there, the tangent space to the zero section, in addition to the canonical vertical
+space which is the kernel of the derivative of the projection), and this is the one that comes
+from our definitions, so this statement is not as crazy as it may seem. In general, avoid using it
+though, since genuinely coordinate-free concepts are always better. -/
```

Comment:
```suggestion
Note that, as currently framed, this is a statement in coordinates, thus reliant on the choice 
of the coordinate system we use on the tangent bundle.

However, the result itself is coordinate-dependent only to the extent that the coordinates
determine a splitting of the tangent bundle.  Moreover, there is a canonical splitting at each
point of the zero section (since there is a canonical horizontal space there, the tangent space 
to the zero section, in addition to the canonical vertical space which is the kernel of the 
derivative of the projection), and this canonical splitting is also the one that comes from the
coordinates on the tangent bundle in our definitions. So this statement is not as crazy as it
may seem.

TODO define splittings of vector bundles; state this result invariantly. -/
```